src/controllers/api.controller.js

```diff
@@ -7,4 +9,34 @@ export default {
 		const filePath = `/img/${req.file.filename}`;
 		res.status(200).json({ responseText: "Uploaded", location: filePath });
 	},
+
+	async getCategories(req, res) {
```

Comment:
It looks like this controller is not used in the middleware, why not reusing it?